### ISSUE TITLE
session restarts

### ISSUE THREAD

**Issue by Jerry-kiwi** [neutrinolabs/xrdp#861]
Target PC: Ubuntu 17.04, xrdp 0.9.3.1, xorgrdp 0.2.3 (built from src)
Client PC: Win 10, Win 7, Ubuntu 17.04 (Remmina 1.2.0-rcgit-20 (git rcgit-20))

When I start RDP session from Linux, I can connect to this session from Win*
But when I start RDP session from Remmina and then try to connect from Win*, it seems, that xrdp daemon is restarted automaticly.

+ i see high CPU usage by XRDP (using htop)

![image](https://user-images.githubusercontent.com/8778463/30027338-2643cbbc-9189-11e7-83f6-6f35af6b5a72.png)

**Comment by jsorg71** [comment]
It looks like you are logging in as user www_adm, that's ok.
What does it mean "start RDP session from Linux" ? Do you mean you use xfreerdp at command line?
Remmina should be using freerdp too.
Do you think the high CPU usage to causes systemd to restart xrdp?
Do do you see the high CPU usage after the restart?

**Comment by Jerry-kiwi** [comment]
1. "start RDP session from _____ "  means, that i reboot target pc, and connect to it using rdp from ____
2. I dont't have ideas about what cause xrdp to restart. 
3. Can you tell, where xrdp store its logs? may be i will help me to find out the reason

Thanks in advance.

**Comment by Jerry-kiwi** [comment]
```
[20170919-07:49:35] [INFO ] A connection received from 127.0.0.1 port 54830
[20170919-07:49:35] [INFO ] ++ reconnected session: username www_adm, display :11.0, session_pid 1931, ip 192.168.10.105:64627 - socket: 12
[20170919-07:49:35] [DEBUG] Closed socket 8 (AF_INET 127.0.0.1:3350)
[20170919-12:27:18] [INFO ] A connection received from 127.0.0.1 port 59398
[20170919-12:27:18] [INFO ] ++ reconnected session: username www_adm, display :11.0, session_pid 1931, ip 192.168.10.105:52661 - socket: 12
[20170919-12:27:18] [DEBUG] Closed socket 8 (AF_INET 127.0.0.1:3350)
[20170919-12:28:11] [CORE ] window manager (pid 1932) did exit, cleaning up session
[20170919-12:28:13] [INFO ] calling auth_stop_session and auth_end from pid 1931
[20170919-12:28:13] [INFO ] ++ terminated session:  username www_adm, display :11.0, session_pid 1931, ip 192.168.10.27:46466 - socket: 12
```

**AT THIS POITN COONECTION WAS NOT KEEPED, BUT TERMINATED, BUT ALL PREVIOUS TIME IT WAS KEPT ALIVE :(**

```
[20170919-15:06:30] [INFO ] A connection received from 127.0.0.1 port 33206
[20170919-15:06:30] [INFO ] ++ created session (access granted): username www_adm, ip 192.168.10.105:58036 - socket: 12
[20170919-15:06:30] [INFO ] starting Xorg session...
[20170919-15:06:30] [DEBUG] Closed socket 9 (AF_INET 0.0.0.0:5912)
[20170919-15:06:30] [DEBUG] Closed socket 9 (AF_INET 0.0.0.0:6012)
[20170919-15:06:30] [DEBUG] Closed socket 9 (AF_INET 0.0.0.0:6212)
[20170919-15:06:30] [DEBUG] Closed socket 8 (AF_INET 127.0.0.1:3350)
[20170919-15:06:30] [INFO ] calling auth_start_session from pid 12737
[20170919-15:06:30] [DEBUG] Closed socket 7 (AF_INET 127.0.0.1:3350)
[20170919-15:06:30] [DEBUG] Closed socket 8 (AF_INET 127.0.0.1:3350)
[20170919-15:06:31] [INFO ] Xorg :12 -auth .Xauthority -config xrdp/xorg.conf -noreset -nolisten tcp -logfile .xorgxrdp.%s.log
[20170919-15:06:31] [CORE ] waiting for window manager (pid 12738) to exit
```
